api/internal/managers/linux/infra/install.go

```diff
@@ -314,6 +319,7 @@ func (m *infraManager) getAddonInstallOpts(license *kotsv1beta1.License, rc runt
 			Namespace:             constants.KotsadmNamespace,
 			ClusterID:             m.clusterID,
 			AirgapBundle:          m.airgapBundle,
+			SkipPreflights:        SkipPreflights,
```

Comment:
isn't v3 an env var? why not just
```suggestion
			SkipPreflights:        os.Getenv("ENABLE_V3") == "1",
```